system/languages/en.php

```diff
@@ -192,6 +192,8 @@
 		"missing-arguments" => "missing arguments",
 		"archive" => "archive",
 		"sure-to-delete" => "Are you sure you want to delete this page",
+		"enter-label" => "Enter label (use a short word)",
```

Comment:
I think it is clearer to use the word Tag here instead of Label. Also it is nice to make the communication with users a bit more human, so I would suggest "Please enter a label/tag" and "Enter a short description". The same goed for the Dutch translation.

Reply:
Do we want to use the word "tag" at all? Or are we going to call these "labels" everywhere anyway (like Github does)?

One alternative could be "Enter label for this tag" (which is what technically happens under the hood - a new tag object is created, which is assigned a label attribute).

Reply:
Conclusion: Let's use "Tag" everywhere, also in the Dutch translation and string ids.